conf/modules.config

```
@@ -70,11 +70,19 @@ if (params.aligner == "alevin") {
             ext.args = "--table transcript_id,gene_id"
             ext.prefix = { "${gff.baseName}_gffread" }
         }
-        withName: 'SALMON_INDEX|SALMON_ALEVIN' {
+        withName: 'SIMPLEAF_INDEX' {
             publishDir = [
                 path: { "${params.outdir}/${params.aligner}" },
                 mode: params.publish_dir_mode
             ]
+            ext.args = "--rlen ${params.simpleaf_rlen}"
```

```suggestion
            ext.args = { "--rlen ${params.simpleaf_rlen}" }
```

just to be sure... if params get overriden in a user config, they might not affect this setting otherwise.

---

Hmmm,

Indeed. Think is better to pass --rlen to the module directly?